lib/services/messenger/translators/discourse.ts

```diff
@@ -356,6 +339,9 @@ export class DiscourseTranslator extends TranslatorScaffold implements Translato
 	): Promise<TranslatorMetadata[]> {
 		return Promise.resolve(_.compact(_.map(response.post_stream.posts, (comment) => {
 			const messageObject = DiscourseTranslator.extractMetadata(comment.cooked, MetadataEncoding.HiddenHTML, config);
+			if (messageObject.hidden || comment.type === 4) {
```

Comment:
Hardcoded value for `comment.type`? Are there any enumerated types we could use?

Reply:
Not that discourse publish, I've added a comment to this effect to make it more obvious why it's a magic number.